src/main/resources/assets/gregtech/lang/en_us.lang

```diff
@@ -4412,7 +4412,8 @@ gregtech.machine.miner.multi.modes=Has Silk Touch and Chunk Aligned Modes.
 gregtech.machine.miner.multi.tooltip=Max Working Area: §b%dx%d§7 chunks.
 gregtech.machine.miner.multi.production=Produces §b3x§7 more crushed ore than a macerator.
 gregtech.machine.miner.multi.description=A multiblock mining machine that covers a large area and produces huge quantity of ore.
-#Small ores lang: gregtech.machine.miner.multi.production=Produces 3x more crushed ore than a macerator. Small ore Fortune §b%d§7.
+#Small ores lang:
+gregtech.machine.miner.multi.production=Produces 3x more crushed ore than a macerator. Small ore Fortune §b%d§7.
```

Comment:
This lang key should stay commented out, ie one line starting with `#` as it was before, as small ores are not implemented at this time.